src/main/java/kitchenpos/domain/OrderRepository.java

```diff
@@ -1,9 +1,15 @@
 package kitchenpos.domain;
 
-import org.springframework.data.jpa.repository.JpaRepository;
-
+import java.util.List;
+import java.util.Optional;
 import java.util.UUID;
 
-public interface OrderRepository extends JpaRepository<Order, UUID> {
+public interface OrderRepository {
+
+    Optional<Order> findById(UUID orderId);
     boolean existsByOrderTableAndStatusNot(OrderTable orderTable, OrderStatus status);
```

Comment:
IntelliJ IDEA의 코드 자동 정렬 기능(⌥⌘L, Ctrl+Alt+L)을 사용하면 더 깔끔한 코드를 볼 수 있을 거예요.

```suggestion
    Optional<Order> findById(UUID orderId);

    boolean existsByOrderTableAndStatusNot(OrderTable orderTable, OrderStatus status);
```